src/pages/profile/index.ts

```diff
@@ -1,24 +1,96 @@
 import Block from '../../utils/Block';
 import template from './profile.hbs';
-import { ProfileSidebarBlock } from '../../blocks/profile_sidebar';
+import { ProfileSidebarBlock,ProfileSidebarBlockProps } from '../../blocks/profile_sidebar';
 import { ProfileAvaBlock } from '../../blocks/profile_ava';
 import { ProfileRowBlock } from '../../blocks/profile_row';
 import { ProfileChangeLinkBlock } from '../../blocks/profile_change_link';
+import { ModalBlock,ModalBlockProps } from '../../blocks/modal';
+import store, { withStore } from '../../utils/Store';
+import AuthController from '../../controllers/AuthController';
+import { Link } from '../../components/link';
+import {Button} from "~src/components/button";
+import {showModal} from "~src/utils/helpers";
 
 interface ProfilePageProps {
-  profileSidebarBlock: ProfileSidebarBlock;
+  profileSidebarBlock: typeof ProfileSidebarBlock;
```

Comment:
если в этом свойстве будет лежать тип, то нужно, чтобы имя свойства об этом говорило
```suggestion
  profileSidebarBlockType: typeof ProfileSidebarBlock;
```